<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmUser</class>
 <widget class="QWidget" name="frmUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>16</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя пользователя:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>44</y>
     <width>77</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Полное имя:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>72</y>
     <width>77</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Пароль:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="UserName">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>16</y>
     <width>169</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="RealUserName">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>44</y>
     <width>169</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>72</y>
     <width>77</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="password2">
   <property name="geometry">
    <rect>
     <x>196</x>
     <y>72</y>
     <width>77</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="Blocked">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>184</y>
     <width>97</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Заблокирован</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butApply">
   <property name="geometry">
    <rect>
     <x>184</x>
     <y>184</y>
     <width>93</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Записать...</string>
   </property>
  </widget>
  <widget class="QComboBox" name="UserRole">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>104</y>
     <width>169</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>104</y>
     <width>93</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Роль:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
